docx/oxml/sdts.py

```diff
@@ -31,10 +31,10 @@ class CT_SdtPr(BaseOxmlElement):
     """
     ``<w:sdtPr>`` represents property element of ``<w:sdt>`` (content control).
     """
-    alias = ZeroOrOne('w:alias')
+    _alias = ZeroOrOne('w:alias')
```

Comment:
now that I've taken a deeper look into their conventions for extracting `val` out of element, yon can see how it's handled on the upstream (they just add `_val` postfix):

https://github.com/python-openxml/python-docx/blob/36cac78de080d412e9e50d56c2784e33655cad59/docx/oxml/table.py#L161
https://github.com/python-openxml/python-docx/blob/36cac78de080d412e9e50d56c2784e33655cad59/docx/oxml/table.py#L785 

Usually, it's a good python practice to keep attributes private and handled over properties (like we tried accomplishing in this pr), but note that this library prefers to have most of the things open for the client user, so i'd suggest keeping up to the existing framework convention.

Reply:
Good catch!